packages/angular/cli/src/commands/completion/cli.ts

```diff
@@ -8,18 +8,57 @@
 
 import { join } from 'path';
 import yargs, { Argv } from 'yargs';
-import { CommandModule, CommandModuleImplementation } from '../../command-builder/command-module';
+import {
+  CommandModule,
+  CommandModuleImplementation,
+  CommandScope,
+} from '../../command-builder/command-module';
+import { addCommandModuleToYargs } from '../../command-builder/utilities/command';
+import { colors } from '../../utilities/color';
+import { initializeAutocomplete } from '../../utilities/completion';
 
 export class CompletionCommandModule extends CommandModule implements CommandModuleImplementation {
   command = 'completion';
   describe = 'Generate a bash and zsh real-time type-ahead autocompletion script.';
   longDescriptionPath = join(__dirname, 'long-description.md');
 
   builder(localYargs: Argv): Argv {
-    return localYargs;
+    return addCommandModuleToYargs(localYargs, CompletionInitCommandModule, this.context);
   }
 
   run(): void {
     yargs.showCompletionScript();
   }
 }
+
+class CompletionInitCommandModule extends CommandModule implements CommandModuleImplementation {
+  command = 'init';
+  describe = 'Set up Angular CLI autocompletion for your terminal.';
+  longDescriptionPath = undefined;
+  static override scope = CommandScope.Both;
```

Comment:
```suggestion
```

Reply:
Maybe I'm not understanding, but I thought the purpose of this `scope` was whether to allow `ng completion init` to run inside a project, outside a project, or both? I think we would want `ng completion init` to work in any context right? It doesn't actually need to be related to a particular project?

Reply:
By default it's both, that why it is redundant.

Reply:
Oh I see. Removed.